scripts/Deploy-OxaStamp.ps1

```diff
@@ -184,7 +190,11 @@ Param(
         
         [Parameter(Mandatory=$false)][switch]$AutoDeploy=$false,
         
-        [Parameter(Mandatory=$false)][string]$EdxAppSecretKey=""
+        [Parameter(Mandatory=$false)][string]$EdxAppSecretKey="",
+        
+        [Parameter(Mandatory=$false)][string]$EdxAppLmsAllowedHosts="",
```

Comment:
Does this equate to allow all by default if not specified?

Reply:
Yes